src/api/handlers.ts

```diff
@@ -0,0 +1,14 @@
+import {AxiosResponse} from "axios";
+
+export class ProsopoApiError extends Error {
+  constructor(error: AxiosResponse, context?: string, ...params: any[]) {
+
+    super(`${error.data.message ? error.data.message : error.statusText}`)
```

Comment:
`data` is returned from the API
https://github.com/prosopo-io/provider/blob/1a0b2f9f9e956b9beaebc14f401ff9e205adef28/src/errors.ts#L169-L203